### ISSUE TITLE
selectedResults is not an array

### ISSUE THREAD

**Issue by gurbirkalsi** [distributed-system-analysis/pbench#1127]
Summary page throws an error when clicking on a row result on the search page. 

```
Summary.componentDidMount
src/pages/Dashboard/Summary.js:94
  91 | } = this.props;
  92 | 
  93 | if (!Array.isArray(selectedResults)) {
> 94 |   throw new Error("selectedResults is not an array!");
     | ^  95 | }
  96 | else if (selectedResults.length <= 0) {
  97 |   throw new Error("no selectedResults!");
```

**Comment by portante** [comment]
@gurbirkalsi, I thought I had fixed this.  Can you add more context?

**Comment by portante** [comment]
@gurbirkalsi, could this issue be referring to the fix made in PR #1136.

**Comment by gurbirkalsi** [comment]
Yes it is! Thanks for catching that @portante

**Comment by portante** [comment]
Fixed by #1136.
